Source/Scene/InvertClassification.js

```diff
@@ -225,9 +193,13 @@ InvertClassification.prototype.update = function (context) {
     }
   }
 
-  if (!defined(this._fbo) || textureChanged || previousFramebufferChanged) {
-    this._fbo = this._fbo && this._fbo.destroy();
-    this._fboClassified = this._fboClassified && this._fboClassified.destroy();
+  if (
+    !defined(this._fbo.framebuffer) ||
+    textureChanged ||
+    previousFramebufferChanged
+  ) {
+    this._fbo.destroyResources();
+    this._fboClassified.destroyFramebuffer();
```

Comment:
There might be a case where the resources aren't recreated if the framebuffer manager update function determines that it's not dirty. Maybe FramebufferManager should consider itself dirty if the framebuffer is undefined.

Also I think `this._fboClassified.destroyFramebuffer();` could just be `this._fboClassified.destroyResources();` since I wouldn't expect this code to be called frequently enough that `fboClassified` would need to hold onto the same color texture.